configurable_lti_consumer/configurable_lti_consumer.py

```
@@ -68,9 +76,13 @@ def __getattribute__(self, item):
         default AND are hidden) and default to the normal behavior which is to retrieve the value
         stored in Mongodb.
         """
+        editable_field_names = getattr(LtiConsumerXBlock, "editable_field_names", None)
+        # Dogwood/eucalyptus compatibility layer
+        if editable_field_names is None:
+            editable_field_names = LtiConsumerXBlock.editable_fields
```

why not:
```python 
getattr(LtiConsumerXBlock, "editable_fields", None)
```

---

Because I want to raise an exception if we need to handle another API. Legit?